runtime/shibuya/src/lib.rs

```diff
@@ -1596,49 +1624,15 @@ pub type Executive = frame_executive::Executive<
 ///
 /// Once done, migrations should be removed from the tuple.
 pub type Migrations = (
-    GovernancePalletsVersionSetting,
```

Comment:
this is not needed with this uplift. It can detect missing storage_version and initialize it 🤷‍♂️

Reply:
That's ok, feel free to remove it.
I remember our discussion  - when the end goal was "only" v1.7.2 🙂